examples/metadata_processing/plot_add_dict_entries.py

```diff
@@ -1,46 +1,36 @@
 """
-=====================================================
-Add dictionary items in the standard DICOM dictionary
-=====================================================
-
-This examples illustrates how to add dictionary items to the 'standard' DICOM
-dictionary dynamically.
-
-.. warning::
-
-   Note that this is not a recommended solution in general but it is useful as
-   a demonstration or for in-house programs only, or to get around elements
-   missing from pydicom's DICOM dictionaries.
-
+=============================================================
+Add private dictionary items to the standard DICOM dictionary
+=============================================================
+
+This examples illustrates how to add private dictionary items to the DICOM
+dictionary dynamically. This allows to add unknown private tags to a new dataset,
```

Comment:
`This allows you to add...`

Reply:
I removed this part as it actually does not work.